src/pages/verify/getting-started/best-security-practices.mdx

```diff
@@ -0,0 +1,97 @@
+---
+product: verify
+category: Getting Started
+sort: 3
+title: OpenID Connect best security practices
+subtitle: Learn about key security guidelines for implementing basic OpenID Connect authentication in client applications.
+---
+
+import Layout from '../../../layouts/mdx';
+
+export default Layout;
+
+In OpenID Connect(OIDC) authentication flows, there are two main roles:
+
+1. **Authorization Server (OpenID Provider)**: the server that authenticates the user and issues ID tokens and access tokens. It handles user authentication and returns the user’s identity information to the client application.
+2. **Client Application**: the application requesting authentication on behalf of the end user.
+
+Criipto Verify is an OpenID Provider, handling most of the process. However, it's important for developers of client applications to also follow security best practices.
+
+## Authorization Code Flow
+
+The most secure method of performing OIDC authentication is the [Authorization Code Flow](/verify/getting-started/oidc-intro/#authenticate-with-back-channel-authorization-code-flow), or its extended version — [Authorization Code Flow with PKCE](http://localhost:8000/verify/getting-started/oidc-intro/#authenticate-with-pkce). 
```

Comment:
`http://localhost:8000` seems off here ?

Reply:
Totally off, thanks for noticing